driver-core/src/test/java/com/datastax/driver/core/MissingRpcAddressTest.java

```diff
@@ -67,10 +67,18 @@ private void deleteNode2RpcAddressFromNode1() throws Exception {
                         Policies.defaultLoadBalancingPolicy(), Lists.newArrayList(firstHost)))
                 .build());
     Session session = cluster.connect();
-    String deleteStmt =
-        String.format(
-            "DELETE rpc_address FROM system.peers WHERE peer = '%s'",
-            ccm().addressOfNode(2).getHostName());
+    String deleteStmt;
+    if (ccm().getCassandraVersion().nextStable().compareTo(VersionNumber.parse("4.0")) < 0) {
+      deleteStmt =
```

Comment:
Shouldn't we instead always delete from system.peers, and additionally, also delete from system.peers_v2 if this table exists?

Reply:
Yes, I think this is correct. I missed that detail when focusing on fixing this for 4.0 as deleting from peers wasn't enough.